Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:18.04
+ARG CESIUM_API_KEY
+ARG MAPBOX_API_ACCESS_TOKEN
+
+
+RUN apt-get update
+RUN apt-get install -y apt-utils
+RUN apt-get install -y python3
+RUN apt-get install -y python3-pip
+RUN apt-get install -y python3-dev
+RUN apt-get install -y sqlite3
+RUN apt-get install -y fftw3
+RUN apt-get install -y libfftw3-dev
+RUN apt-get install -y git
+RUN apt-get -y upgrade
+
+RUN pip3 install --system pipenv
+
+RUN echo PATH="$HOME/.local/bin:$PATH" >> ~/.bashrc
+RUN /bin/bash -c "source ~/.bashrc"
+
+RUN git clone https://github.com/PX4/flight_review.git
+
+WORKDIR "flight_review"
+
+## update caesium and mabox keys
+RUN sed -i "/cesium_api_key =/ s/=.*/=$CESIUM_API_KEY/" config_default.ini
```

Comment:
Can we create a `config_user.ini` for these?

Reply:
👍🏻